discussions/distributed-text-services-api.md

```diff
@@ -0,0 +1,126 @@
+# Distributed Text Services API
+
+Status: nascent.  Not yet ready for implementation.
+
+## Media Types
+
+The media type for DTS is JSON-LD, using Hydra conventions [http://www.hydra-cg.com](http://www.hydra-cg.com). The `@id` of any resource can be any URI, which may be a URL or a URI. Document content can be in any media type, including TEI, HTML, PDF, syntax trees, or image formats.
+
+## HTTP Status Codes
+
+Standard conventions for HTTP Status Codes are used.  No custom status codes are allowed.
+
+## Entry Point
+
+A client needs to know the URL of the entry point, which is determined by the server. In this example, the entry point is `/dts/api`, but a server can choose a different relative URL. All other resources can be discovered via navigation or queries from the entry point.  URLs shown in payloads in this document may differ from the URLs provided by a given server, and should not be relied on by clients.
+
+If the client does a `GET` on the entry point URL, the following document is returned.
+
+```
+{
+  "@context": "/dts/api/contexts/EntryPoint.jsonld",
+  "@id": "/dts/api/",
+  "@type": "EntryPoint",
+  "collections": "/dts/api/collections/",
+  "documents": "/dts/api/documents/",
+  "navigation" : "/dts/api/navigation",
+  "search" : "/dts/api/search"  
+}
+```
+
+Note: This document does not address which services are optional.  That will be determined by the group.
+
+## Collections
+
+The collections entry point is used for navigating collections. A collection contains metadata for the collection itself and an array of members.  Each member is either a collection or the metadata for a document.
+
+The hierarchy of collections is not fixed.  One server might provide all documents in a flat collection or a collection hierarchy organized by geography, time, or any other convenient logical grouping.
+
+### Hydra Representation and Hierarchy
+
+DTS does not specify any particular hierarchy of collections. A collection might provide all documents in a flat collection or a collection hierarchy organized by geography, time, or any other convenient logical grouping.
+
+A server that uses the CTS collection hierarchy might provide the following top level collection:
+
+```
+{
+  "@context": "http://www.w3.org/ns/hydra/context.jsonld",
+  "@id": "urn:perseus:collections",
+  "@type": "Collection",
+  "totalItems": "2",
+  "member": [
+    {
+      "@id": "urn:perseus:greekLit"
+      "title" : "Ancient Greek"
+    },
+    {
+      "@id": "urn:perseus:latinLit",
+      "title": "Classical Latin"
+    },
+    ...
+  ]
+}]
+```
+
+
+### Query Parameters
+
+The collections endpoint supports the following query parameters:
+
+| name | description                              | methods |
+|------|------------------------------------------|---------|
+| id   | identifier for a collection or document. |  GET    |
+| find | keyword / value search on collection or document metadata       |  GET    |
```

Comment:
If we include find here, why would it not be included elsewhere? Feel like this should be consistent throughout the api

Reply:
This comes from old discussions and was not kept for the current draft.